### PR TITLE
feat(playground): add ability to expand code block by default

### PR DESCRIPTION
**Pull request by amandaejohnston**
This PR adds the `expandCodeByDefault` prop to the playground. The prop is also set on the button example at URL `/docs/playground/button` for testing purposes.

### CODE REVIEW

File: src/components/global/Playground/index.tsx [hunk 1]
```diff
@@ -92,13 +92,15 @@ export default function Playground({
...
   devicePreview,
+  defaultCodeExpanded = false,
 }: {
   code: { [key in UsageTarget]?: MdxContent | UsageTargetOptions };
   title?: string;
   src: string;
   size: string;
   description?: string;
   devicePreview?: boolean;
+  defaultCodeExpanded: boolean
```
[the remark this answers is not in the document]
Comment (amandaejohnston) [reply src/components/global/Playground/index.tsx:103]: How about `expandCodeByDefault`? Too wordy? I want to make it clear that it's only used as the default, rather than anything tied to the active state.

[the remark this answers is not in the document]
Comment (liamdebeasi) [reply src/components/global/Playground/index.tsx:103]: Works for me 👍
